### PR TITLE
Select fitness-related POIs.

### PR DESCRIPTION
**Pull request by zerebubuth**
This selects fitness-related POIs and normalises them to `kind=fitness`.

Connects to #465. Requires mapzen/TileStache#110.

@nvkelso could you review, please?

### CODE REVIEW

File: queries.yaml [hunk 1]
```diff
@@ -112,6 +112,7 @@ layers:
     transform:
       - TileStache.Goodies.VecTiles.transform.tags_create_dict
       - TileStache.Goodies.VecTiles.transform.add_iata_code_to_airports
+      - TileStache.Goodies.VecTiles.transform.normalize_leisure_kind
```
Comment (nvkelso) [inline queries.yaml]: If we're normalizing them all to `fitness` should this be `normalize_fitness_kind` instead?

File: data/functions.sql [hunk 2]
```diff
@@ -178,6 +178,10 @@ BEGIN
       WHEN amenity_val  = 'ski_rental'       THEN LEAST(zoom + 1.27, 17)
       WHEN amenity_val  = 'ski_school'       THEN LEAST(zoom + 2.30, 15)
       WHEN man_made_val = 'snow_cannon'      THEN LEAST(zoom + 4.90, 18)
+      WHEN leisure_val  = 'sports_centre'    THEN LEAST(zoom + 3.98, 17)
+      WHEN leisure_val  = 'fitness_centre'   THEN LEAST(zoom + 3.98, 17)
+      WHEN leisure_val  = 'fitness_station'  THEN LEAST(zoom + 3.98, 17)
+      WHEN amenity_val  = 'gym'              THEN LEAST(zoom + 3.98, 17)
```
Comment (nvkelso) [inline data/functions.sql]: Should `gym` be normalized to `fitness` as well?

File: queries.yaml (hunk 1 above)
Comment (zerebubuth) [inline queries.yaml]: Depends on whether we want to write another function the next time we normalise another leisure kind or extend the existing one. I don't have a strong feeling either way - do you prefer `normalize_fitness_kind`?

File: data/functions.sql (hunk 2 above)
Comment (zerebubuth) [inline data/functions.sql]: Yes, `amenity=gym` should become `kind=gym` when coalesced in the query, and the transform should normalise that to `kind=fitness` [on this line](https://github.com/mapzen/TileStache/pull/110/files#diff-52de7615e270521ccb11d3a0fe443d33R2713).

Comment (nvkelso) [inline data/functions.sql]: Looks good, tnx.

File: queries.yaml (hunk 1 above)
Comment (nvkelso) [inline queries.yaml]: General is better, leave as is.

**Comment by nvkelso** [comment]
Ready to merge.

**Pull request merged**